### PR TITLE
feat: add currency to wds table

### PR DESCRIPTION
**Pull request by KelvinOm**
## Description
- Copy currency functionality of Table widget to WDS. [Original PR](https://github.com/appsmithorg/appsmith/pull/27819).
- Fix imports

### CODE REVIEW

File: app/client/src/widgets/wds/WDSTableWidget/widget/propertyConfig/PanelConfig/Basic.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ValidationTypes } from "constants/WidgetValidation";
 import type { TableWidgetProps } from "widgets/TableWidgetV2/constants";
-import { ICON_NAMES } from "widgets/TableWidgetV2/constants";
+import { ColumnTypes, ICON_NAMES } from "widgets/TableWidgetV2/constants";
```
Comment (KelvinOm) [inline app/client/src/widgets/wds/WDSTableWidget/widget/propertyConfig/PanelConfig/Basic.ts:3]: I'll fix the imports by a separate PR

**Review by dhruvikn — APPROVED** [review]

**Review by sbalaji1192 — APPROVED** [review]
Looks good!

**Pull request merged**